File: 10474.cpp
#include <iostream>
using namespace std;
#include <vector>
#include <algorithm>
int main()
{
    int n, q;
    int time = 1;
    while (cin >> n >> q)
    {
        if (n == 0 && q == 0)
            break;
        vector<int> v(n);
        for (int i = 0; i < n; i++)
        {
            cin >> v[i];
        }
        cout << "CASE# " << time << ":" << endl;
        sort(v.begin(), v.end());
        int k;
        for (int i = 0; i < q; i++)
        {
            cin >> k;
            int j;

            for (j = 0; j < v.size(); j++)
            {
                if (v[j] == k)
                {

                    cout << k << " found at " << (j + 1) << endl;
                    break;
                }
            }
            if (j == v.size())
            {
                cout << k << " not found " << endl;
            }
        }
        time++;
    }
}

File: 10474.py
time = 1

while True:
    n, q = map(int, input().split())
    if n == 0 and q == 0:
        break
    v = [int(input()) for _ in range(n)]
    v.sort()
    print(f'CASE# {time}:')
    for _ in range(q):
        k = int(input())
        for i in range(n):
            if v[i] == k:
                print(f'{k} found at {i + 1}\n')
                break
        else:
            print(f'{k} not found\n')
       
        

    time += 1
